backend/de.metas.postfinance/src/main/java/de/metas/postfinance/PostFinanceConstants.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * #%L
+ * de.metas.postfinance
+ * %%
+ * Copyright (C) 2024 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.postfinance;
+
+public interface PostFinanceConstants
+{
+	String XML_PROPERTY_VALUE_YES = "yes";
+
+	String CUSTOMER_REGISTRATION_MESSAGE = "R";
+
+	String CUSTOM_FIELD_CBPARTNERVALUE = "CBPARTNERVALUE";
```

Comment:
i know it's a bit annoying, but please let's change this to `BPARTNEREXTERNALREFERENCE` (max 35chars, just uppercase-chars and numbers)..or something else, but "value" is missleading.

Reply:
done. Will ask PostFinance to upload another file with this customer field instead of `CBPARTNERVALUE`